boards/phytec/phyboard_pollux/doc/index.rst

```diff
@@ -110,6 +114,33 @@ created by NXP and can be found at
 :zephyr_file:`dts/arm/nxp/nxp_imx8ml_m7.dtsi`. The Pinout of the phyBOARD-Polis
 can be found at the `PHYTEC website`_.
 
+I2C
+---
+
+There are multiple on-device I2C devices already connected to
+I2C busses.
+Some of these devices should not be used by the M7-Core if
+running the PHYTEC BSP on the A53-Core.
+
+Here is an overview of the on-device I2C devices:
+
++-----------------+-----------------+--------------------+-------------------+
+| I2C Device      | Address         | Usage              | Can be used by M7?|
++=================+=================+====================+===================+
+| PMIC            | 0x25@i2c1       | Power Management   |Should not be used |
++-----------------+-----------------+--------------------+-------------------+
+| EEPROM          | 0x51@i2c1       | EEPROM             |Should not be used |
+| (Atmel 24C32)   |                 | (U-Boot config)    |                   |
++-----------------+-----------------+--------------------+-------------------+
+| RTC (RV3028)    | 0x51@i2c1       | Real Time Clock    | yes               |
```

Comment:
This should be marked as `Should not be used`. I2C1 has the PMIC connected and should be used exclusively by the A53 cores. I don't think the A53 and M7 can access this bus in parallel.